File: benchmark/sum.py
import time

i = 0
total = 0

t0 = time.perf_counter()
while i < 1_000_000:
    total += (i * 3) % 97 - (i % 7)
    i += 1

print((time.perf_counter() - t0) * 1000)

File: benchmark/sum.lua
local i = 0
local total = 0

local t0 = os.clock()
while i < 1000000 do
    total = total + (i * 3) % 97 - (i % 7)
    i = i + 1
end

print((os.clock() - t0) * 1000)
